integration-tests/tasks/konflux-e2e-tests-task.yaml

```diff
@@ -50,8 +54,8 @@ spec:
           cpu: "500m"
           memory: "1Gi"
         limits:
-          cpu: "2"
-          memory: "6Gi"
+          cpu: "4"
+          memory: "8Gi"
```

Comment:
do we really need that [based on your observations](https://redhat-internal.slack.com/archives/C02FANRBZQD/p1722495750888019?thread_ts=1722242471.349299&cid=C02FANRBZQD)?